conda-forge.yml

```diff
@@ -1,4 +1,6 @@
 build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
```

Comment:
As cross-compiling requires more work, let's save this for PR ( https://github.com/conda-forge/faiss-split-feedstock/pull/62 ) where it is attempted

```suggestion
```